sphinx/util/template.py

```diff
@@ -84,6 +84,8 @@ def __init__(self, template_path: str = None, latex_engine: str = None) -> None:
         self.env.variable_end_string = '%>'
         self.env.block_start_string = '<%'
         self.env.block_end_string = '%>'
+        self.env.comment_start_string = '<#'
+        self.env.comment_end_string = '<#'
```

Comment:
Is this not wrong? Should `self.env.comment_end_string` not be `#>`?

Reply:
Oops... @yves-chevallier Please make a new PR :-)